index.html

```diff
@@ -1151,6 +1155,40 @@ <h2>Constrainable Properties</h2>
         </tbody>
       </table>
     </section>
+    <section>
+      <h2>Capture timestamps</h2>
+      <div>
+          Some video sources can supply information about when a video frame was captured.
+          This information is useful for example for AV sync and end-to-end delay measurement.
+        </p>
+        <p>
+          {{VideoFrameMetadata}}.{{VideoFrameMetadata/captureTime}} MUST be set by the user
+          agent in video frames sourced from {{MediaDevices/getUserMedia()}},
+          {{MediaDevices/getDisplayMedia()}}, and {{HTMLMediaElement/captureStream()}}.
```

Comment:
What about {{Canvas/captureStream}}?

What if an HTMLMediaElement's srcObject is a MediaStream containing a video MediaStreamTrack from gUM() or gDM()? Does HTMLMediaElement/captureStream() generate a new timestamp or carry forward the original?

Reply:
Also, this doesn't say what user agents should do for other sources. Is it to leave always the member out? Or try to reconstitute it, if known somehow?

Reply:
The idea is to define in what way each source set the captureTime following recommendations in comments of https://github.com/w3c/webcodecs/pull/813, so this is just one of multiple PRs to make it fit together.